docs/source/markdown/podman-container-inspect.1.md

```diff
@@ -18,6 +18,48 @@ all results in a JSON array. If a format is specified, the given template will b
 Format the output using the given Go template.
 The keys of the returned JSON can be used as the values for the --format flag (see examples below).
 
+Valid placeholders for the Go template are listed below:
+
+| **Placeholder**      | **Description**                                    |
+| -----------------    | ------------------                                 |
+| .AppArmorProfile     | AppArmor profile (string)                          |
```

Comment:
Is this (string) (array of strings) (etc) useful?

Reply:
I think it is unless it becomes overwhelming to the man page.